Improving Codespaces (#2600)
akaszynski/pyansys: .devcontainer/codespaces-docs/welcome.rst

> ===========================================================
> Welcome to GitHub PyMAPDL Codespaces for Documentalist!! 📚
> ===========================================================

Reviewer: ```suggestion
=========================================================
Welcome to GitHub PyMAPDL Codespaces for Documentalist 📚
=========================================================
```
Avoid exclamation points in technical doc. Is "Documentalist" correct? I'm not familiar with this term (yet).

Author: I don't know... whe I wrote "documentalist" I was thinking in people like you.. who works with docs constantly.. they dont need to test the library.. but build their docs.

Author: the `devcontainer-local` and the one at the root (`codespaces-dev`) is for developer... no docs dependencies are installed (although you could).

🛑 Disclaimer
=============

> You **must use this Codespace only for developing, contributing, documenting and/or
> write examples for PyMAPDL**.

Reviewer: ```suggestion
**You must use this Codespace only for developing, contributing, documenting, and
writing examples for PyMAPDL**.
```
It seemed weird to have everything but the subject in bold.

> Additionally, this Codespace is using an MAPDL Student version which is
> available `here <https://www.ansys.com/academic/students>`_

Reviewer: ```suggestion
Additionally, this Codespace is using an MAPDL Student version. For information on how Ansys
supports your learning Ansys software, see the Ansys for Students <https://www.ansys.com/academic/students>`_ page on the Ansys website.
```
Goggle dev style guide says to never use "here" for a link but the name of the title or page that the link takes you to.

> This version has some limitations.
> For more information visit
> `Ansys MAPDL Student <https://www.ansys.com/academic/students/ansys-student>`_

Reviewer: ```suggestion
The Ansys Student version has some limitations.
For more information, see the
`Ansys Student - Free Software Download <https://www.ansys.com/academic/students/ansys-student>`_
 page on the Ansys website.```

📖 Codespace configuration
==========================

This Codespace has been set for helping developers to document and write examples

> in PyMAPDL in the easiest way. This Codespace contains all the OS and Python dependencies
> for running PyMAPDL and build the documentation in HTML and PDF.

Reviewer: ```suggestion
in PyMAPDL in the easiest way. It contains all the OS and Python dependencies
for running PyMAPDL and building the documentation in HTML and PDF.
```

Some of the available Python dependencies are:

> * Numpy
> * Pandas
> * Pyvista
> * Sphinx
> * plotly

Reviewer: ```suggestion
* NumPy
* Pandas
* PyVista
* Sphinx
* Plotly
```
We should match the case that the originating sources use in their documentation.

🧐 How to

> =========

Reviewer: ```suggestion
==========
```

> To start documenting PyMAPDL, it is recommended you visit
> `Write documentation <https://mapdl.docs.pyansys.com/version/dev/getting_started/write_documentation.html>`_
> where the steps needed are detailed.

Reviewer: ```suggestion
Before you start documenting PyMAPDL, see
`Write documentation <https://mapdl.docs.pyansys.com/version/dev/getting_started/write_documentation.html>`_,
which describes the necessary steps.
```

> You can see the latest documentation regarding using Codespaces with PyMAPDL
> `here <https://mapdl.docs.pyansys.com/version/dev/getting_started/devcontainer_link.html>`_.

Reviewer: ```suggestion
You can see the latest documentation on using Codespaces with PyMAPDL in
`Develop on a container <https://mapdl.docs.pyansys.com/version/dev/getting_started/devcontainer_link.html>`_.
```
Consider adding a target and using the :ref: role rather than a ULR to the HTML page?

Author: I would totally do that (as the rest of the document) but this file `welcome.rst` is not included in the docs, it is the file opened by the codespace when starting up... so adding target will obscure the link and make it less useful. :)

😊 Finally
==========

> We hope you enjoyed this Codespace! 

Reviewer: ```suggestion
We hope you enjoy this Codespace. 
```
I'll let the exclamation point in line 53 remain.

Author: I appreciate a lot the concession! :P xD

**Happy coding! 💻**

See also
========

> * `Learning PyMAPDL <https://mapdl.docs.pyansys.com/version/dev/getting_started/learning.html>`_. 

Reviewer: Again consider using the :ref: role for these links as they are to topics within the same guide.

Author: Same reasons as given in: https://github.com/ansys/pymapdl/pull/2600/files#r1442190173 :)
